demos/motion_tracking/dvs_file_input/process.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import numpy as np
+from numpy.lib.stride_tricks import as_strided
+from scipy import signal
+from dv import AedatFile
```

Comment:
How are we going to deal with these added dependencies, `dv`, `bokeh`, possibly others, if this is demo is going to be part of lava-dnf?

Reply:
We should add the additionally required packages in the description of the demo, so users can install them on their own. Metavision_sdk and possible other sensor/device specific packaged need a similar solution anyways.

Reply:
Done.